include/mxnet/kvstore.h

```diff
@@ -361,6 +373,17 @@ class KVStore {
    */
   virtual void SendCommandToServers(int cmd_id, const std::string& cmd_body) { }
 
+  /**
+   * \brief Sends server profiler commands to all server nodes
+   * Only the worker with rank=0 sends the command which will be received by all servers
+   * \param type ProfilerCommand type
+   * \param params parameters for that command in the form of a string
+   */
+  virtual void SetServerProfilerCommand(const KVStoreServerProfilerCommand type,
+                                        const std::string& params) {
+    LOG(FATAL) << "compile with USE_DIST_KVSTORE=1 to use distributed kvstore";
```

Comment:
do you really need to die here? will a warning suffice?

Reply:
or a CHECK may be a caught error, right? is FATAL catchable? python user might not know whether kvstore was turned on. i mean, is this something where execution just can’t continue?

Reply:
There's nothing we can do here. The server can't be started if the above case is true

Reply:
yeah but will the local still run?

Reply:
what I mean is, for builds that may not have kvstore enabled, you might still want code to work, but only locally and not just blow up

Reply:
In the case of local kvstore, there is no concept of a server, so there is no second process to which we would need to send params. But I could add a warning that this call is useless, as there is no second process when kvstore is not distributed